sdk/identity/azure-identity-broker/README.md

```diff
@@ -46,7 +46,7 @@ To take dependency on a particular version of the library that isn't present in
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-identity-broker</artifactId>
-    <version>1.1.8</version>
+    <version>1.1.9</version>
```

Comment:
Shouldn't this point to version `1.1.10`? Or is it intentionally a version behind?

Reply:
Same - it's dependency, not current, so it'll get fixed after current becomes dependency.